cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MetricsMessageCallback.java

```diff
@@ -41,6 +43,8 @@
 public final class MetricsMessageCallback implements KafkaConsumer.MessageCallback {
 
   private static final Logger LOG = LoggerFactory.getLogger(MetricsMessageCallback.class);
+  private static final ImmutableMap<String, String> METRICS_PROCESSOR_CONTEXT =
+    ImmutableMap.of(Constants.Metrics.Tag.NAMESPACE, "system", Constants.Metrics.Tag.COMPONENT, "metrics.processor");
```

Comment:
better to Constants.SYSTEM_NAMESPACE and another constant for metrics-processor. would be easier to track.